GhostLegs/glw169/bones_sol.py

```diff
@@ -40,9 +39,6 @@
 # ASCI art input
 start_points = list(zip(starting_line.split(), [i*3 for i in range(w//2)]))
```

Comment:
Why `i*3`? Why `w//2`? Explain use of `//` vs `/`, and where these constants come from.

Reply:
Simplified associated code into a more clear, concise one-line solution.